app/controllers/application_controller.rb

```diff
@@ -1,3 +1,12 @@
+# frozen_string_literal: true
+
+# Main controller
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  # After user successfuly logged in, redirect to Main page, accessible only for logged in users
+  def after_sign_in_path_for(user)
+  	#binding.pry
```

Comment:
this commend is not very helpful, so it should not be in PR

Reply:
Would try to take make function names more meaningful instead of using comments